src/p2G4_main.c

```diff
@@ -161,6 +192,7 @@ static void f_rx_search(uint d) {
 
     bs_trace_raw_time(8,"RxDone (NoSync during Rx_search) for device %u\n", d);
 
+    rx_status->state = Rx_State_Inactive;
```

Comment:
Not a functional problem, but just something which may be misleading: Many will end not going back to `Rx_State_Inactive` in many cases but be left in found.
May it be better to change `state` to be just searching/not_searching?

Reply:
I originally had that but for the corner case where two transmitters start at the same time the current behavior of lowest device number wins would be altered. If thats no an issue then a boolean would work.

Reply:
Uhm, a change like this:
```diff
diff --git a/src/p2G4_main.c b/src/p2G4_main.c
index 5f9b3fa..2bf8e05 100644
--- a/src/p2G4_main.c
+++ b/src/p2G4_main.c
@@ -96,7 +96,7 @@ static void find_and_activate_rx(const p2G4_tx_t *tx_s, uint tx_d) {
            (rx_status->rx_s.radio_params.modulation & P2G4_MOD_SIMILAR_MASK))
     {
       rx_status->tx_nbr = tx_d;
-      rx_status->state = Rx_State_Found;
+      rx_status->state = Rx_State_NotSearching;
       fq_add(current_time, Rx_Found, rx_d);
     }
   }
@@ -192,7 +192,7 @@ static void f_rx_search(uint d) {
 
     bs_trace_raw_time(8,"RxDone (NoSync during Rx_search) for device %u\n", d);
 
-    rx_status->state = Rx_State_Inactive;
+    rx_status->state = Rx_State_NotSearching;
     rx_status->rx_done_s.end_time = current_time;
 
     p2G4_phy_resp_rx(d, &rx_status->rx_done_s);
diff --git a/src/p2G4_pending_tx_rx_list.h b/src/p2G4_pending_tx_rx_list.h
index 398b3c4..19e99a2 100755
--- a/src/p2G4_pending_tx_rx_list.h
+++ b/src/p2G4_pending_tx_rx_list.h
@@ -77,9 +77,8 @@ void txl_activate(uint dev_nbr);
  * Reception state
  */
 typedef enum {
-    Rx_State_Inactive = 0, //Not actively searching for any tx
+    Rx_State_NotSearching = 0, //Not actively searching for any tx
     Rx_State_Searching,//Searching for a fitting tx
-    Rx_State_Found,//Found a fitting tx and actively trying to receive it
 } rx_state_t;
```
Should not change the execution flow in any way (You still take it out of searching in `find_and_activate_rx()` )